core/src/main/scala-2/spire/syntax/Ops.scala

```diff
@@ -421,6 +421,7 @@ final class RightModuleOps[V](x: V) {
   def :*[F](rhs: Int)(implicit ev: RightModule[V, F], F: Ring[F]): V = ev.timesr(x, F.fromInt(rhs))
 }
 
+@deprecated
```

Comment:
I think it would be better UX if these deprecation annotations had an accompanying short message. The same goes for all other places.